src/utils.js

```diff
@@ -664,10 +664,10 @@ module.exports.isNumber = function(value) {
 };
 
 /**
- * Strips zero width chars and padding spaces from the displayName
+ * Strips zero width chars, diaritics and whitespace from the displayName
```

Comment:
I think you mean "diacritics"

Reply:
Whoops, I rely on my IDE's spellcheck kicking in too heavily and it somehow didn't catch that :(